### PR TITLE
inubekit-fieldset: Create component `<Fieldset/>` 

### PR DESCRIPTION
**Pull request by cmarin001**
This pull request introduces the development and integration of a new `<Fieldset/>` component into our application's UI library. Aimed at enhancing form organization and usability, the `<Fieldset/>`  component provides developers with a standardized way to group related form elements, improving the overall user experience in form interactions.

### CODE REVIEW

File: src/Fieldset/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Text } from "@inubekit/text";
+import { Stack } from "@inubekit/stack";
+import { useMediaQuery } from "@inubekit/hooks";
+import { StyledFieldset } from "./styles";
+import { Size, TypographyType } from "./props";
+
+interface FieldsetProps {
```
Comment (wfercanas) [inline src/Fieldset/index.tsx:7]: Definamos la interfaz sólo como `IFieldset`, no me di cuenta en qué momento cambiamos este estándar en los componentes.

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/Fieldset/props.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  docs: {
+    description: {
+      component:
+        "This component helps us to be able to separate content and title it.",
+    },
+  },
+};
+
+const props = {
+  title: {
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/Fieldset/props.ts:20]: solved

File: src/Fieldset/index.tsx (hunk 1 above)
Comment (cmarin001) [reply src/Fieldset/index.tsx:7]: solved

File: src/Fieldset/typography.ts
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/Fieldset/typography.ts:1]: solved

File: src/Fieldset/index.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  return (
+    <StyledFieldset isMobile={isMobile}>
+      <legend>
+        <Stack padding="4px">
+          <Text type={type} size={size}>
+            {title}
+          </Text>
+        </Stack>
+      </legend>
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/Fieldset/index.tsx:27]: solved

**Review by wfercanas — APPROVED** [review]

**Pull request merged**